src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -66,13 +66,13 @@
       FROM
         (SELECT
           fuzzer,
-          IF(strategy_corpus_mutations_radamsa > 0, "radamsa,", "") AS s_radamsa,
-          IF(strategy_random_max_len > 0, "max len,", "") AS s_max_len,
-          IF(strategy_corpus_mutations_ml_rnn > 0,"ml rnn,", "") AS s_ml_rnn, 
-          IF(strategy_value_profile > 0, "value profile,", "") AS s_vp, 
+          IF(strategy_corpus_mutations_radamsa > 0, "corpus_mutations_radamsa,", "") AS s_radamsa,
+          IF(strategy_random_max_len > 0, "random_max_len,", "") AS s_max_len,
+          IF(strategy_corpus_mutations_ml_rnn > 0,"corpus_mutations_ml_rnn,", "") AS s_ml_rnn, 
+          IF(strategy_value_profile > 0, "value_profile,", "") AS s_vp, 
           IF(strategy_fork > 0, "fork,", "") AS s_fork,
-          IF(strategy_corpus_subset > 0, "subset,", "") AS s_subset,
-          IF(strategy_recommended_dict > 0, "dict,", "") AS s_recommended_dict,
+          IF(strategy_corpus_subset > 0, "corpus_subset,", "") AS s_subset,
+          IF(strategy_recommended_dict > 0, "recommended_dict,", "") AS s_recommended_dict,
```

Comment:
Just a question: how easy will it be to add more strategies here? I'll enable a new one in https://github.com/google/clusterfuzz/pull/583, should I update this file and some other places?